src/index.js

```diff
@@ -0,0 +1,147 @@
+// --------------------------------wave3 and 6 Naming the City----------------------
+const cityNameLabel = document.getElementById('cityname');
+const cityNameInput = document.getElementById('cityname-input');
+const resetBtn = document.getElementById('reset-btn');
+// change city name on top header
+const addCityName = document.getElementById('topCityName');
+var currCity;
+// handle Get Realtime Temperature button to check weather
+const gettemperature = document.getElementById('gettemperature');
+var count;
+
+cityNameInput.addEventListener('change', (event) => {
+  currCity = event.target.value;
+  console.log('change:', currCity);
+  cityNameLabel.innerText = currCity;
+  addCityName.innerText = currCity;
+});
+resetBtn.addEventListener('click', () => {
+  cityNameInput.value = '';
+  cityNameLabel.innerText = 'City Name';
+});
+
+gettemperature.addEventListener('click', () => {
+  findLocation(currCity);
+});
+// --------------------------------------wave4 Calling APIs-------------------
+const findLocation = (query) => {
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: query,
+        format: 'json',
+      },
+    })
+    .then((response) => {
+      latitude = response.data[0].lat;
+      longitude = response.data[0].lon;
+      console.log(response);
+      console.log('success in findLatitudeAndLongitude!', latitude, longitude);
+      // make the next API call here!
+      // findcity(latitude, longitude);
+      getWeather(latitude, longitude);
+    })
+    .catch((error) => {
+      console.log('error in findLatitudeAndLongitude!');
+    });
```

Comment:
Nice job with the multiple API calls! As an alternative can you consider how this would work with promise chaining?